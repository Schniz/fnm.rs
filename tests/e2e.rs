#[macro_use]
mod shellcode;

mod feature_tests;
