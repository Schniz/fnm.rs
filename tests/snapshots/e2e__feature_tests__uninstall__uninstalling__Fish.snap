---
source: tests/e2e.rs
expression: "&source.trim()"
---
fnm env | source
fnm install v10.11.0
fnm uninstall v10.11.0 | grep 'Uninstalled version'
fnm use v10.11.0 2>&1 | grep 'is not currently installed'
